dts/bindings/pinctrl/nxp,mcux-rt-pinctrl.yaml

```diff
@@ -0,0 +1,143 @@
+# Copyright (c) 2021 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  The node has the 'pinctrl' node label set in MCUX RT SoC's devicetree. These
+  node can be autogenerated using the MCUXpresso config tools combined with
+  the parse-rt-mex.py script in the NXP HAL.
+
+
+compatible: "nxp,mcux-rt-pinctrl"
+
+include:
+    - name: base.yaml
+    - name: pincfg-node-group.yaml
+      child-binding:
+        property-allowlist:
+          - drive-open-drain
+
+properties:
+    reg:
+      required: true
+
+child-binding:
+  description: |
+    MCUX RT pin controller pin configuration group.
+  child-binding:
+    properties:
+      pinmux:
+        required: true
+        type: array
+        description: |
+          An array of values defining the configuration register for a given
+          pinmux mode. See fsl_iomuxc.h in the NXP HAL for a list.
+      nxp,software_input_on:
+        required: false
+        type: int
+        default: 0
+        enum:
+          - 0
+          - 1
+        description: |
+          00 Pin input follows peripheral
+          01 Pin is forced to input
+      nxp,hysteresis_enable:
```

Comment:
may `input-schmitt-enable` be suitable? Again, it should be a boolean

Reply:
as user, when I see below two options
```
      input-schmitt-enable:
        required: false
        type: boolean
        description: enable schmitt-trigger mode

      input-schmitt-disable:
        required: false
        type: boolean
        description: disable schmitt-trigger mode
```
I am quite confused on this, and I prefer to keep this as RT user manual name this fucntion.